.github/ISSUE_TEMPLATE/ feature_request copy.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
this file has a space in the beginning of the name, and a "copy"? Mind renaming it?

Reply:
OMG! that is totally wrong!